words_n_fun/preprocessing/api.py

```diff
@@ -171,6 +171,22 @@ def get_preprocessor(pipeline: list = DEFAULT_PIPELINE, prefered_column: str = '
                         chunksize=chunksize, first_row=first_row, columns=columns, sep=sep,
                         nrows=nrows, **pandas_args)
 
+@utils.data_agnostic
+def process_block_of_data(chunk: pd.Series, pipeline: list, max_chunksize: int):
```

Comment:
There is no possibility anymore via this function to use tqdm, you should add this possibility